app/views/shared/_identity.html.erb

```diff
@@ -30,6 +30,14 @@
           <% end %>
         </a>
       </li>
+      <%
+        if @context && @context.is_a?(Course)
+          resources_module = @context.context_modules.where("name ILIKE '%Braven Resources%'").first
+          unless resources_module.nil?
+      %>
+            <li><%= link_to "Resources", "/courses/#{@context.id}/modules/#{resources_module.id}" %></li>
```

Comment:
Can we add a CSS class or ID on this in case we want to style or perform some logic to hide it in certain scenarios?